awesomemotive/affiliatewp-affiliate-area-shortcodes
# Issue/48 - Fixed banner message not showing when AffiliateWP is not installed

Issue #48 

Steps to test:

1) Install Affiliate Shortcodes add-on;

You should see a banner message like the image below telling you that the add-on requires AffiliateWP:

![image](https://user-images.githubusercontent.com/116665983/210437199-6f377e90-b2f5-434a-8b20-d47f34ca24ce.png)

(Observation: the message changes if the AffiliateWP is listed as a plugin or not)

2) Now install AffiliateWP, the message should go away.

![image](https://user-images.githubusercontent.com/116665983/210437445-c1b94d90-a7e3-4826-a87f-2a8099918a6b.png)



**Housekeeping**


**Proposed changes**
